src/foremast/runner.py

```diff
@@ -111,7 +111,7 @@ def create_pipeline(self, onetime=None):
                 spinnakerpipeline = pipeline.SpinnakerPipelineS3(**kwargs)
             elif pipeline_type == 'datapipeline':
                 spinnakerpipeline = pipeline.SpinnakerPipelineDataPipeline(**kwargs)
-            elif pipeline_type == 'manual':
+            elif pipeline_type == 'manual' or pipeline_type in consts.MANUAL_PIPELINE_SYNONYMS:
```

Comment:
Could we simplify this to just:

`elif pipeline_type in consts.MANUAL_PIPELINES` and put the value of `"manual"` in the list?

Reply:
In an earlier check do an and check for `ALLOWED_MANUAL_TYPES`